scripts/release/shared-commands/print-prerelease-summary.js

```diff
@@ -22,14 +22,7 @@ module.exports = ({cwd}) => {
 
     {header Before publishing, please smoke test the packages!}
 
-    1. Open {path ./fixtures/packaging/babel-standalone/dev.html} in the browser.
-    2. It should say {quote "Hello world!"}
-    3. Next go to {path ./fixtures/packaging} and run {command node build-all.js}
-    4. Go to the repo root and {command npx pushstate-server . 9000}
-    5. Open {link http://localhost:9000/fixtures/packaging}
-    6. Verify every iframe shows {quote "Hello world!"}
-
-    After completing the above steps, you can publish this release by running:
```

Comment:
We don't need to do this manually anymore! 😁

Reply:
🙏